python-sdk/nuscenes/utils/splits.py

```diff
@@ -212,6 +214,16 @@ def create_splits_scenes(verbose: bool = False) -> Dict[str, List[str]]:
 
     return scene_splits
 
+def get_prediction_challenge_split(split: str) -> List[str]:
+    if split not in {'mini_train', 'mini_val', 'train', 'val'}:
+        raise ValueError("split must be one of (mini_train, mini_val, train, val)")
+
+    path_to_file = os.path.join(os.path.dirname(os.path.realpath(__file__)), "prediction_scenes.json")
```

Comment:
Are the scenes handpicked or simply all train/val sub-scenes?

Reply:
They are subsets of the existing train/val split.